layouts/partials/js.html

```diff
@@ -1,8 +1,16 @@
-<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-<script src="{{ .Site.BaseURL }}js/scrollreveal.min.js"></script>
-<script type='application/javascript' src='{{ .Site.BaseURL }}js/smooth-scrollbar.js'></script>
-<script src="{{ .Site.BaseURL }}js/baffle.min.js"></script>
-<script src="{{ .Site.BaseURL }}js/main.js"></script>
+<script>
+(function () {
+  if (window.fetch) return;
+  document.write('<script src="https://cdn.jsdelivr.net/jquery/3.2.1/jquery.slim.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/scrollreveal.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/smooth-scrollbar.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/baffle.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/baffle.min.js"><\/script>');
+  document.write('<script src="{{ .Site.BaseURL }}js/main.js"><\/script>');
+})()
+</script>
+
+<script defer src="{{ .Site.BaseURL }}js/main-jquery.js"></script>
```

Comment:
This was my lame attempt at getting the jQuery stuff not to throw an error. Should be totally unnecessary to split this stuff up. But it's not a terrible pattern if trying to put a tourniquet on jQuery in favor of Web Standards libs.